packages/rancher-kiali-server/package.yaml

```diff
@@ -1,4 +1,4 @@
-url: https://kiali.org/helm-charts/kiali-server-1.35.0.tgz
```

Comment:
> I had to temporarily track two rancher-kiali-server versions, but this will be reverted when I complete the 1.10.4 backport

why do you need two rancher-kiali-server versions at the same time? why not update the existing one to 1.35.1?

Reply:
Normally for a new version I would update it if possible, but since we are doing an OOB release I want to introduce a new istio version and nothing else.

Reply:
Also its only temporary until the version is moved to the release branch. I will be cleaning up the repo afterwards to only have 1 kiali version.

Reply:
ok, it makes sense to me to have a temporary Kiali chart. 
My next question is why do you need to change the existing Kiali package?

Reply:
Mainly it was my attempt to "not make any changes to the existing kaili in production" - so I renamed the existing package to rancher-kiali-server-1.35 in the dev branch so that I could check out the 1.32 version from the commit history at the 1.9.6 deployment commit. That way I knew that the rancher-kiali-server package I was using for istio 1.9.8 is the same one that was used for the istio currently in production. 

The rancher-kiali-server package I renamed has not been released yet for 2.5.x release so it made sense that that would be the one to temporarily change.  

If it would make it easier to reivew, I could revert this PR https://github.com/rancher/charts/pull/1381 and then redo this one after the revert gets merged.

Reply:
Thanks for the explanation, it is clear to me now. 
As a note to other reviewers:
- the `packages/rancher-kiali-server-1.35` is the original `packages/rancher-kiali-server`
- the `packages/rancher-kiali-server` is modified to be Istio 1.32 and used for Istio 1.9.8
- the `packages/rancher-kiali-server-1.35` will be moved back to `packages/rancher-kiali-server`

Reply:
per arvind's suggestion, we are no longer tracking two kiali versions.